docs/gen_cli_docs.py

```
@@ -0,0 +1,660 @@
+    ]
+    return '\n'.join(lines)
+
+
+# ---------------------------------------------------------------------------
+# Main
+# ---------------------------------------------------------------------------
+
+def _git_hash(path):
+    try:
+        r = subprocess.run(
+            ['git', 'rev-parse', '--short', 'HEAD'],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)